backend/candidate/views/my_views.py

```
@@ -1836,7 +1824,7 @@ def upload_resume_modal(request):
                 if not request.user.mobile and mobile:
                     request.user.mobile = mobile
                     
-            except Exception as e:
+            except Exception:
                 # Continue even if text extraction fails
                 pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)